reference/sessions.qmd

```diff
@@ -86,17 +86,23 @@ Tuesday June 24: 11.00-12.30
     - joint estimation of delays, nowcasts and reproduction numbers (35 mins)
 -   Wrap up (10 mins)
 
-## Session 8: Forecasting concepts
+## Session 8: Visualising infectious disease forecasts
```

Comment:
do we actually want to keep this called forecast concpets and rename the session? It feels more like forecasting concepts than vis imo?

Reply:
yep sounds good